api/room/models.py

```diff
@@ -41,12 +42,12 @@ class Room(Base, Utility):
         order_by="func.lower(Resource.name)")
     events = relationship('Events', cascade="all, delete-orphan")
     response = relationship('Response', cascade="all, delete-orphan")
-    devices = relationship('Devices', cascade="all, delete-orphan")
     room_tags = relationship(
         'Tag',
         secondary="room_tags",
         backref=('tags'),
         lazy="joined")
+    devices = relationship('Devices', cascade="all, delete-orphan")
```

Comment:
Kindly add an `order_by func` on this to be able to sort the devices alphabetically

Reply:
Done @nakatuddesuzan